[2022-01-04 23:37:58,741] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-01-04 23:37:58,752] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-01-04 23:37:58,753] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-04 23:37:58,753] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-04 23:37:58,753] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-04 23:37:58,865] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): also_run_this> on 2022-01-04 00:00:00+00:00
[2022-01-04 23:37:58,867] {standard_task_runner.py:52} INFO - Started process 11484 to run task
[2022-01-04 23:37:58,872] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', '/home/rhavel/Documentos/DEPLOY-PROJECTS/first-airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpmgywzck0', '--error-file', '/tmp/tmpimv6zjwe']
[2022-01-04 23:37:58,873] {standard_task_runner.py:77} INFO - Job 6: Subtask also_run_this
[2022-01-04 23:37:59,449] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2022-01-04T00:00:00+00:00 [running]> on host rhavel-VPCEG13EB
[2022-01-04 23:37:59,693] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-01-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-04T00:00:00+00:00
[2022-01-04 23:37:59,694] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-04 23:37:59,694] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2022-01-04T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-01-04 00:00:00+00:00: scheduled__2022-01-04T00:00:00+00:00, externally triggered: False>"']
[2022-01-04 23:37:59,703] {subprocess.py:85} INFO - Output:
[2022-01-04 23:37:59,705] {subprocess.py:89} INFO - run_id=scheduled__2022-01-04T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2022-01-04 00:00:00+00:00: scheduled__2022-01-04T00:00:00+00:00, externally triggered: False>
[2022-01-04 23:37:59,705] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-04 23:37:59,797] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220104T000000, start_date=20220105T023758, end_date=20220105T023759
[2022-01-04 23:37:59,928] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-04 23:37:59,967] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
